scripts/update-all-remote-wn-clients

```diff
@@ -64,6 +77,10 @@ def call_updater(cfg, section, updater_script, log_dir, dry_run=False):
     if remote_dir:
         cmd.append("--remote-dir=%s" % remote_dir)
 
+    if timeout:
+        cmd.insert(0, "timeout")
+        cmd.insert(1, str(timeout))
+
```

Comment:
Where does the timeout in the command call actually get consumed?

Reply:
This changes the program being run to `/usr/bin/timeout` which takes a duration and another command to run.